src/lumigo_tracer/spans_container.py

```diff
@@ -243,6 +246,18 @@ def get_tags_len(self) -> int:
     def add_tag(self, key: str, value: str) -> None:
         self.function_span[EXECUTION_TAGS_KEY].append({"key": key, "value": value})
 
+    def start_manual_trace(self, name: str) -> None:
+        now = int(datetime.now().timestamp() * 1000)
+        self.manual_trace_timers[name] = now
+
+    def stop_manual_trace(self, name: str) -> None:
+        manual_trace_started = self.manual_trace_timers.pop(name, None)
+        if manual_trace_started:
+            now = int(datetime.now().timestamp() * 1000)
+            self.function_span[MANUAL_TRACES_KEY].append(
```

Comment:
I would actually put it in the `self.function_span[MANUAL_TRACES_KEY]` already when it starts.
It will help you to identify the spans that causes timeouts.
This function should only update the `endTime`

Reply:
Actually it will not work because the function_span will not be ended in this case, which is a *real* bummer.
Why didn't you consider it as a span?

Reply:
Its a good idea but its not supported yet, I will keep it as-is